src/applications/vaos/utils/eligibility.js

```diff
@@ -11,16 +11,24 @@ import {
   getClinics,
 } from '../api';
 
-export async function getEligibilityData(facilityId, typeOfCareId, systemId) {
+export async function getEligibilityData(
+  facilityId,
+  typeOfCareId,
+  systemId,
+  isDirectScheduleEnabled,
+) {
   const eligibilityChecks = [
     checkPastVisits(facilityId, typeOfCareId, 'request'),
     getRequestLimits(facilityId, typeOfCareId),
-    checkPastVisits(facilityId, typeOfCareId, 'direct'),
-    getClinics(facilityId, typeOfCareId, systemId),
   ];
 
-  if (typeOfCareId === PRIMARY_CARE) {
-    eligibilityChecks.push(getPacTeam(facilityId));
+  if (isDirectScheduleEnabled) {
```

Comment:
I moved the check here because we want to stop the checks from being run and it also messes with the overall logic if it looks like you can make a DS appointment on the facility page, but we change that later.